src/api/app/models/buildresult.rb

```diff
@@ -39,6 +39,16 @@ class Buildresult < ActiveXML::Node
     unknown:      'The scheduler has not yet evaluated this package. Should be a short intermediate state for new packages.'
   }.with_indifferent_access.freeze
 
+  def self.find_hashed(opts = {})
```

Comment:
[Nitpick]

This is something that to me it feels that belongs to that second level (not implemented yet) in the `Backend::*` library, that one that reads from the backend a string, then parses it with one XML library and returns a basic type for user's consumption.